messaginginapp/src/main/java/io/customer/messaginginapp/ui/bridge/InAppHostViewDelegate.kt

```diff
@@ -0,0 +1,50 @@
+package io.customer.messaginginapp.ui.bridge
+
+import android.view.ViewGroup
+import androidx.core.view.isVisible
+import io.customer.messaginginapp.gist.presentation.engine.EngineWebView
+
+/**
+ * Delegate interface that abstracts host view operations for in-app messages.
+ * Allows operations like adding/removing child views, creating new view instances,
+ * and posting actions on UI thread without directly depending on Android framework.
+ * Designed for easier testing and mocking.
+ */
+internal interface InAppHostViewDelegate {
+    var isVisible: Boolean
+
+    fun addView(delegate: EngineWebViewDelegate)
+    fun removeView(delegate: EngineWebViewDelegate)
+    fun createEngineWebViewInstance(): EngineWebViewDelegate
+    fun post(action: () -> Unit)
```

Comment:
Same as above, reused same names from `View` class in this delegate to avoid reimplementation and keep the overall implementation simpler.